Intermediate Level 📁/README.md

```diff
@@ -1,8 +1,8 @@
 <table width="100%">
            <tr>
              <td align="center">
-               <img src="https://github.com/Kushal997-das/Hacktoberfest-2024/blob/master/Intermediate%20Level%20%F0%9F%93%81/Kushal997-das/my%20github%20profile.JPG" height="250" width="300"/>
-               <p><a href="https://github.com/Kushal997-das">Kushal Das</a></p>
+               <img src="./kumar007ambi/my github profile.JPG" height="250" width="300"/>
+               <p><a href="https://github.com/kumar007ambi">Ambika Kumar Kewat</a></p>
             </td>
           </tr>
 </table>
```

Comment:
@kumar007ambi You have removed the existing content and added yours, which is not acceptable. This is the final reminder; otherwise, I will close this pull request.

Reply:
Hi @Kushal997-das I have undo the Readme.md as it was,please take a look